src/rsa/public/exponent.rs

```diff
@@ -77,6 +78,18 @@ impl Exponent {
 
         Ok(Self(value))
     }
+
+    /// The big-endian encoding of the exponent.
+    ///
+    /// There are no leading zeros.
+    pub fn be_bytes(&self) -> impl ExactSizeIterator<Item = u8> + Clone + '_ {
```

Comment:
FYI, before merging this, I removed `#[inline]` from here. It doesn't make to inline this now that it is so complicated.

Reply:
Thanks for the heads up. I generally don't pay too much attention to the `#[inline]` attribute because rustc and LLVM are already extremely aggressive about inlining.